test/consts.test.ts

```diff
@@ -49,4 +49,20 @@ describe('consts', () => {
             }
         });
     });
+
+    describe('ACTOR_ENV_VARS', () => {
+        it('every value begins with "ACTOR_"', () => {
+            Object.values(ACTOR_ENV_VARS).forEach((v) => {
+                expect(v.startsWith('ACTOR_')).toBe(true);
+            });
+        });
+    });
+
+    describe('APIFY_ENV_VARS', () => {
+        it('is the same as ENV_VARS', () => {
+            Object.keys(APIFY_ENV_VARS).forEach((k) => {
+                expect(APIFY_ENV_VARS[k]).toBe(ENV_VARS[k]);
+            });
+        });
+    });
```

Comment:
Here we also can be stricter
```suggestion

        it('every value is "APIFY_" + key', () => {
            Object.entries(APIFY_ENV_VARS).forEach(([k, v]) => {
                // TODO: remove this once ACTOR_MAX_PAID_DATASET_ITEMS is removed from APIFY_ENV_VARS
                if (k === 'ACTOR_MAX_PAID_DATASET_ITEMS') return;

                expect(v).toBe(`APIFY_${k}`);
            });
        });
    });
```